### PR TITLE
Metricbeat: Replace ceph image from demo to daemon

### PR DESCRIPTION
**Pull request by jsoriano**

### CODE REVIEW

File: metricbeat/module/ceph/_meta/Dockerfile [hunk 1]
```diff
@@ -11,3 +8,8 @@ HEALTHCHECK --interval=1s --retries=300 \
         | jq .output.health.health_services[0].mons[0] \
         | grep health
 EXPOSE 5000
+
+ENV NETWORK_AUTO_DETECT 4
```
Comment (jsoriano) [inline metricbeat/module/ceph/_meta/Dockerfile]: I couldn't make it work with static config as we had before, but it seems to work with this.

File: metricbeat/module/ceph/_meta/Dockerfile [hunk 2]
```diff
@@ -11,3 +8,8 @@ HEALTHCHECK --interval=1s --retries=300 \
...
+ENV DEMO_DAEMONS osd,rest_api
```
Comment (jsoriano) [inline metricbeat/module/ceph/_meta/Dockerfile]: New image starts a UI in the same port as the rest_api, preventing the rest_api to work, I'm not sure what is the daemon starting the UI, but with this set of daemons the api starts correctly.

**Comment by exekias** [comment]
jenkins, retest this please

**Comment by jsoriano** [comment]
Thanks to @andrewvc for start investigating the issue!

**Comment by exekias** [comment]
failing test is unrelated to this change

**Pull request merged**